File: iniciante/python/1000/1036.py
entry = input().strip().split()
A = float(entry[0])
B = float(entry[1])
C = float(entry[2])

delta = B**2 - 4 * A * C

if A == 0 or delta < 0:
    print("Impossivel calcular")
else:
    x1 = (-B + pow(delta, 1/2)) / (2*A)
    x2 = (-B - pow(delta, 1/2)) / (2*A)
    print(f'R1 = {x1:.5f}')
    print(f'R2 = {x2:.5f}')

File: iniciante/c/1000/1036.c
#include <stdio.h>
#include <math.h>

int main(){
    double A, B, C;
    scanf("%lf %lf %lf", &A, &B, &C);
    double delta = B*B - 4 * A * C;
    if(A == 0 || delta < 0){
        printf("Impossivel calcular\n");
    } else{
        printf("R1 = %.5lf\n", ((-B + sqrt(delta)) / (2*A)));
        printf("R2 = %.5lf\n", ((-B - sqrt(delta)) / (2*A)));
    }
    return 0;
}